src/backend/distributed/executor/multi_utility.c

```diff
@@ -164,6 +164,40 @@ static void PostProcessUtility(Node *parsetree);
 static bool warnedUserAbout2PC = false;
 
 
+void
+multi_ProcessUtility(Node *parsetree,
+					 const char *queryString,
+					 ProcessUtilityContext context,
+					 ParamListInfo params,
+					 DestReceiver *dest,
+					 char *completionTag)
+{
+	PlannedStmt *plannedStmt = makeNode(PlannedStmt);
+	plannedStmt->commandType = CMD_UTILITY;
+	plannedStmt->utilityStmt = parsetree;
+
+	multi_ProcessUtility10(plannedStmt, queryString, context, params, NULL, dest,
```

Comment:
Renamed the old function to 9x to reflect its use by the 9.x-line of PostgreSQL releases. The logic now simply lives in `multi_processUtility`.